### PR TITLE
[zh-cn]: update translation of CSS `border-width` property

### PR DESCRIPTION
**Pull request by T34-active**

### CODE REVIEW

File: files/zh-cn/web/css/border-width/index.md [hunk 1]
```diff
@@ -1,90 +1,116 @@
...
+{{CSSInfo}}
+
+## 形式语法
+
+{{csssyntax}}
 
 ## 示例
 
-### A mix of values and lengths
+### 值与 length 的组合
```
Comment (T34-active) [inline files/zh-cn/web/css/border-width/index.md:81]:
```suggestion
### 值与 length 的混合
```

File: files/zh-cn/web/css/border-width/index.md [hunk 2]
```diff
@@ -1,90 +1,116 @@
...
 ```
 
-### 正式语法
+`border-width` 属性可以通过一个、两个、三个或四个值来指定。
 
-{{csssyntax}}
+- 当指定**一个**值时，该宽度将应用于**四个角**。
+- 当指定**两个**值时，第一个宽度应用于**顶部和底部**，第二个宽度应用于**左侧和右侧**。
+- 当指定**三个**值时，第一个宽度应用于**顶部**, 第二个宽度应用于**左侧和右侧**, 第三个宽度应用于**底部**.
+- 当指定**四个**值时，这些宽度按照**顶部**、**右侧**、**底部**和**左侧** 的顺序（顺时针）进行应用。
```
Comment (T34-active) [inline files/zh-cn/web/css/border-width/index.md:57]:
```suggestion
- 当指定**四个**值时，这些宽度按照**顶部**、**右侧**、**底部**和**左侧**的顺序（顺时针）进行应用。
```

File: files/zh-cn/web/css/border-width/index.md [hunk 3]
```diff
@@ -1,90 +1,116 @@
...
+l10n:
+  sourceCommit: 42c1bb8c259f3f57de9f38600776cf273e3addda
 ---
 
 {{CSSRef}}
 
-## 概要
-
-**`border-width`** 属性可以设置盒子模型的边框宽度。
+**`border-width`** [CSS](/zh-CN/docs/Web/CSS) 是一个用于设置元素边框的宽度[简写属性](/zh-CN/docs/Web/CSS/Shorthand_properties)。
```
Comment (skyclouds2001) [inline files/zh-cn/web/css/border-width/index.md:10]:
```suggestion
**`border-width`** [CSS](/zh-CN/docs/Web/CSS) 是一个用于设置元素边框宽度的[简写属性](/zh-CN/docs/Web/CSS/Shorthand_properties)。
```

File: files/zh-cn/web/css/border-width/index.md [hunk 4]
```diff
@@ -1,90 +1,116 @@
...
+l10n:
+  sourceCommit: 42c1bb8c259f3f57de9f38600776cf273e3addda
 ---
 
 {{CSSRef}}
 
-## 概要
-
-**`border-width`** 属性可以设置盒子模型的边框宽度。
+**`border-width`** [CSS](/zh-CN/docs/Web/CSS) 是一个用于设置元素边框宽度的[简写属性](/zh-CN/docs/Web/CSS/Shorthand_properties)。
```
Comment (jasonren0403) [inline files/zh-cn/web/css/border-width/index.md:10]:
```suggestion
**`border-width`** [CSS](/zh-CN/docs/Web/CSS) [简写属性](/zh-CN/docs/Web/CSS/Shorthand_properties)用于设置元素边框的宽度。
```

File: files/zh-cn/web/css/border-width/index.md [hunk 5]
```diff
@@ -1,90 +1,116 @@
...
+border-width: revert;
+border-width: revert-layer;
+border-width: unset;
 ```
 
-### 正式语法
+`border-width` 属性可以通过一个、两个、三个或四个值来指定。
 
-{{csssyntax}}
+- 当指定**一个**值时，该宽度将应用于**四个角**。
```
Comment (jasonren0403) [inline files/zh-cn/web/css/border-width/index.md:54]:
```suggestion
- 当指定**一个**值时，该宽度将应用于**四条边**。
```

**Review by jasonren0403 — COMMENTED** [review]
一些建议

**Review by jasonren0403 — APPROVED** [review]
👍

**Pull request merged**